docs-source/content/userguide/managing-domains/fmw-infra/_index.md

```diff
@@ -0,0 +1,343 @@
+If you plan to experiment with upper stack products (which are not officially supported
+by the operator yet) we strongly recommend using the domain on a persistent
+volume approach. 
+
+#### Limitations
+
+Compared to running a WebLogic Server domain in Kubernetes using the operator, the
+following limitations currently exist for FMW Infrastructure domains: 
+
+* The [WebLogic Logging Exporter](https://github.com/oracle/weblogic-logging-exporter)
+  currently only supports WebLogic server logs.  Other logs will not be sent to
+  Elasticsearch.  Note however that you can use a sidecar with a log handling tool
+  like Logstash or fluentd to get logs. 
+* The [WebLogic Monitoring Exporter](https://github.com/oracle/weblogic-monitoring-exporter)
+  currently only supports the WebLogic MBean trees.  Support for JRF MBeans has not
+  been added yet.
+* Only configured clusters are supported.  Dynamic clusters are not supported for
+  FMW Infrastructure domains.  Note that you can still use all of the scaling features,
+  you just need to define the maximum size of your cluster at domain creation time.
+* FMW Infrastructure domains are not supported with any version of the operator
+  before version 2.2.
+
+#### Creating a FMW Infrastructure Docker image
+
+You will need to create a Docker image containing the FMW Infrastructure binaries.
+A [sample](https://github.com/oracle/docker-images/tree/master/OracleFMWInfrastructure)
+is provided in the Oracle GitHub account that demonstrates how to create a Docker image
+to run FMW Infrastructure.  
+
+After cloning the repository and downloading the installer from Oracle Technology Network
+or e-delivery, you would create your image by running the provided script:
+
+```bash
+cd docker-images/OracleFMWInfrastructure/dockerfiles
+./buildDockerImage.sh -v 12.2.1.3 -g 
+```
+
+The image produced will be named `oracle/fmw-infrastructure:12.2.1.3`.
+
+You must also install the [required patch]({{< relref "/userguide/introduction/introduction/_index.md#prerequisites" >}})
+to use this image with the operator.  A [sample](https://github.com/oracle/docker-images/tree/master/OracleFMWInfrastructure/samples/12213-patch-fmw-for-k8s)
+is provided that demonstrates how to create a Docker image with the necessary patch installed.
+
+After downloading the patch from My Oracle Support, you can create the patched image
+buy running the provided script:
+
+TODO TODO TODO check with monica's sample to ensure this is correct TODO TODO TODO
+
+```bash
+cd docker-images/OracleFMWInfrastructure/samples/12213-patch-fmw-for-k8s
+./buildDockerImage.sh 
+```
+
+This will produce an image named `TODO TODO TODO monica what is it called TODO TODO TODO`.
+
+These sample will allow you to create a Docker image containing the FMW Infrastructure
+binaries and the necessary patch.  You can use this image to run Repository Creation Utility
```

Comment:
to run Repository Creation Utility -> to run the Repository Creation Utility

Reply:
fixed